e2e/pageobjects/ide/ContextMenu.ts

```diff
@@ -0,0 +1,39 @@
+import 'reflect-metadata';
+import { injectable, inject } from 'inversify';
+import { DriverHelper } from '../../utils/DriverHelper';
+import { CLASSES } from '../../inversify.types';
+import { WebElement, Button, By, Key} from 'selenium-webdriver';
+import { TestConstants} from '../../TestConstants';
+
+
+
+@injectable()
```

Comment:
Broken formatting in file.

Reply:
There are absent line before `async invokeContextMenuOnActiveElementWithKeys()` method, and redundant line after it.